cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestResponse.java

```diff
@@ -44,6 +51,11 @@ public InputStream getStream() {
         return body.byteStream();
     }
 
+    public Map<String, String> getCookies() {
```

Comment:
Recommend to implement a cookie interface with minimal api to support what you need. It will allow for future expansion if needed while only having what you need now

Reply:
Added a cookie interface and a wrapper implementation